pkgs/by-name/si/simutrans/package.nix

```diff
@@ -202,15 +445,19 @@ let
         places. Simutrans is an open source remake of Transport Tycoon.
       '';
 
-      homepage = "http://www.simutrans.com/";
+      homepage = "https://www.simutrans.com/";
       license = with lib.licenses; [
         artistic1
         gpl1Plus
       ];
-      maintainers = [ ];
+      maintainers = with lib.maintainers; [
+        sodiboo
+        Makuru
+      ];
       platforms = lib.platforms.linux; # TODO: ++ darwin;
```

Comment:
We should set sourceProvenance

Reply:
hm.. i'm actually not sure if the paksets contain source code or bytecode. will find out

Reply:
source code. the Squirrel language does have a JIT compiler and from what i gather it supports an intermediate bytecode representation (like a jar file), but the game seems to expect *source code* in the paksets.

`fd '\.nut$' $(nix build --print-out-paths --no-link .#simutrans)`